lib/src/installer/completion_installation.dart

```diff
@@ -90,6 +90,11 @@ class CompletionInstallation {
     }
   }
 
+  /// Define the [File] in which the completion configuration is stored.
+  File get completionConfigurationFile {
+    return File(path.join(completionConfigDir.path, 'config.json'));
+  }
```

Comment:
When is this relevant other than for instantiating `CompletionConfiguration`? whay dont return `CompletionConfiguration` instead?

Reply:
This is a great suggestion. I did think about this but I kept it as it because I thought it would obscure the fact that the **immutable** `CompletionConfiguration` is in fact **immutable**, and that "getting" it doesn't guarantee the same instance to be returned since it is derived from the content of a file. Since you brought it up I think doing the change doesn't obscure this fact, I'll ammend.

```dart
CompletionConfiguration get completionConfiguration {
    return CompletionConfiguration.fromFile(File(path.join(completionConfigDir.path, 'config.json'));)
}
```

Reply:
A consideration is on `lib/src/command_runner/completion_command_runner.dart`, which seems that it is the reason for this to be a public method, which is logic that shouldn't be there. 

This can be a private getter as well as the file path only, as long as this doesnt leak to `CommandRunner`